docs/Examples.md

```diff
@@ -504,6 +504,30 @@ You can also install apps hosted remotely using URLs:
         - "https://example.com/splunk-apps/app3.tgz"
 ```
 
+Also these application configuration parameters can be placed in a `defaults.yml`
+file and use the `defaultsUrlApps` parameter.  The `defaultsUrlApps` parameter
+is specific for applicastion installation and will install the apps in the
+correct instances as per the deployment.
+
+Unlike `defaultsUrl` which is applied at every instance created by the CR, the
+`defaultsUrlApps` will be applied on instances that will **not** get the application
+installed via a bundle push.  earch head and indexer cluster members will not have
+the `defaultsUrlApps` parameter applied.  This means:
+
+ - For standalones the apps will be installed as normal.
+ - For SHC these applications will be installed in the SHC Deployer and push it out
```

Comment:
Nitpick: Just so that both statements are consistent
 - For SHC, these applications will only be installed on the SHC Deployer and pushed to the members via SH Bundle push.

Reply:
fixed